src/tv2-common/content/dve.ts

```diff
@@ -416,6 +409,39 @@ export function MakeContentDVE2<
 	}
 }
 
+function createLocatorsTimelineObjects(
+	config: TV2ShowStyleConfig,
+	dveConfig: DVEConfigInput,
+	graphicsTemplateContent: { [key: string]: string },
```

Comment:
Just curious as you hinted at it yourself: Why is `{ [key: string]: string }` used over `Record<string, string>`? :D